doc-content/enterprise-only/openshift/secrets-secrettarget-create-proc.adoc

```diff
@@ -0,0 +1,62 @@
+[id='secrets-{secrettarget}-create-proc']
+ifeval::["{secrettarget}"=="kie"]
+:secrettarget_verbose: {KIE_SERVER}
+:appsecret_name: kieserver-app-secret
+endif::[]
+ifeval::["{secrettarget}"=="central"]
+:secrettarget_verbose: {CENTRAL}
+:appsecret_name: {CENTRAL_ONEWORD}-app-secret
+endif::[]
+ifeval::["{secrettarget}"=="smartrouter"]
+:secrettarget_verbose: Smart Router
+:appsecret_name: smartrouter-app-secret
+endif::[]
+ifeval::["{secrettarget}"=="employeerostering"]
+:secrettarget_verbose: the Employee Rostering application
+:appsecret_name: optaweb-employee-rostering-app-secret
+endif::[]
+
+:CAN_AUTOCREATE!:
+ifeval::["{context}"=="openshift-ansible-playbook"]
+:CAN_AUTOCREATE:
+endif::[]
+ifeval::["{context}"=="openshift-operator"]
+:CAN_AUTOCREATE:
+endif::[]
+= Creating the secrets for {secrettarget_verbose}
+
+ifeval::["{secrettarget}"=="kie"]
+OpenShift uses objects called _secrets_ to hold sensitive information such as passwords or keystores. For more information about OpenShift secrets, see the https://access.redhat.com/documentation/en-us/openshift_container_platform/3.11/html/developer_guide/dev-guide-secrets[Secrets chapter] in the OpenShift documentation.
+endif::[]
+
+ifndef::CAN_AUTOCREATE[]
+You must create an SSL certificate for HTTP access to {secrettarget_verbose} and provide it to your OpenShift environment as a secret.
+endif::CAN_AUTOCREATE[]
+ifdef::CAN_AUTOCREATE[]
+In order to provide HTTPS access, {secrettarget_verbose} uses an SSL certificate. The deployment can create a sample secret automatically. However, in production environments you must create an SSL certificate for {secrettarget_verbose} and provide it to your OpenShift environment as a secret.
+endif::CAN_AUTOCREATE[]
+
+ifeval::["{secrettarget}"=="central"]
+Do not use the same certificate and keystore for {CENTRAL} and {KIE_SERVER}.
+endif::[]
+ifeval::["{secrettarget}"=="smartrouter"]
+Do not use the same certificate and keystore for Smart Router as the ones used for {KIE_SERVER} or {CENTRAL}.
+endif::[]
+
+.Procedure
+. Generate an SSL keystore with a private and public key for SSL encryption for {KIE_SERVER}. For more information on how to create a keystore with self-signed or purchased SSL certificates, see https://access.redhat.com/documentation/en-US/JBoss_Enterprise_Application_Platform/6.1/html-single/Security_Guide/index.html#Generate_a_SSL_Encryption_Key_and_Certificate[Generate a SSL Encryption Key and Certificate].
++
+[NOTE]
+====
+In a production environment, generate a valid signed certificate that matches the expected URL for {secrettarget_verbose}. 
+====
++
+. Save the keystore in a file named `keystore.jks`. 
+. Record the name of the certificate. The default value for this name in {PRODUCT} configuration is `jboss`.
```

Comment:
Just curious if it's common/correct to say "in {PRODUCT} configuration" vs. "in a {PRODUCT} configuration" or something. 

Also, not in this PR:
* In the custom image section, maybe change "It pushes this custom image ..." to "The procedure pushes this custom image ..." or something. Best to avoid "it" when possible, especially as the subject of a new sentence.
* Same section, step 2, suggest "... Software Downloads page of the Red Hat Customer Portal."
* Same section, step 3, "...unzipped file. This directory ..." (capitalize)
* Same, step 5, "... provided. For example:" No go for IBM, should be complete, like "provided, as illustrated in the following example:" like you've done in the snippet that follows it. Speaking of, should that one say "...illustrated in the following example:" (singular)?
* In the GlusterFS section, for step one, would "To see if" be better than "To check whether"? I just wonder if "check" is too casual or not globally accessible. Trivial.
* Search "For example:" throughout and decide if you want to address in compliance with IBM or deliberately deviate from the convention. I leave the choice to you.

Reply:
With "for example" my idea is to comply when possible but deviate when necessary. And in this case deviation is necessary, because "...provided, as illustrated in the following example" would be incorrect. However, I changed to "Examples:"; I think I will use *this* form when deviating.

Reply:
Made the change everywhere except autogenerated files.

Reply:
I did not understand the request about Step 3.

Reply:
"In product configuration" - I'm referring to the process/defaults not any particular configuration

Reply:
For step 3, it has the typo "unzipped file. this directory". Cap "This"